tai64/Cargo.toml

```diff
@@ -13,14 +13,15 @@ keywords    = ["tai64", "time", "timestamps", "chrono"]
 
 [badges]
 maintenance = { status = "passively-maintained" }
-travis-ci = { repository = "iqlusioninc/crates", branch = "develop" }
 
 [dependencies]
 chrono = { version = "0.4", optional = true, default-features = false }
+serde =  { version = "*", optional = true, default-features = false }
```

Comment:
Crates without specific version requirements for dependencies can't be published. This needs to be changed to `version = "1"`